frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -36,11 +32,23 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
 
   const basePath = resourcePath(getVMLikeModel(vmiLike).kind, name, namespace);
   const DisksTitle = React.useCallback(
-    ({ children }) => <Link to={`${basePath}/${VM_DETAIL_DISKS_HREF}`}>{children}</Link>,
+    ({ children }) => (
+      <Link
+        to={`${basePath}/${VM_DETAIL_DISKS_HREF}?rowFilter-disk-types=${DiskType.DISK.getValue()}`}
+      >
+        {children}
+      </Link>
+    ),
     [basePath],
   );
   const CDROMTitle = React.useCallback(
```

Comment:
IMO it would be also good to consider hiding stuff in inventory when the count is 0

Reply:
nice, hide on count < 1 :-)